api/controller/contestants.go

```diff
@@ -36,23 +36,25 @@ func GetSingleContestant(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	contestants, err := fetchContestansFromCSV(resp, w)
+	contestants, err := fetchContestansList(resp, w)
 	if err != nil {
 		fmt.Println(err)
 	}
 
+	fmt.Println("Endpoint reached: /contestants/" + strconv.Itoa(id))
+
 	for i := range contestants {
 		if contestants[i].ID == id {
-			resp.JSON(w, http.StatusOK, contestants[i])
+			con := contestants[i]
+			con.Bio = utils.FetchInfo(id)
```

Comment:
This call to the api is redundant isn't it? Since in your `fetchContestantsList` function you're already populating this field